neuroinfer/code/BayesianAnalysis.py

```diff
@@ -80,15 +85,18 @@ def run_bayesian_analysis(cog_list, prior_list, x_target, y_target, z_target, ra
             total_database_nq = len(set(paper_id_nq))
             total_not_cog_nq = total_database_nq - len(ids_cog_nq)
 
-            lik_cog_nq = round(intersection_cog_nq / len(ids_cog_nq), 3)
```

Comment:
Come mai 3 o 4? Ne va di precisione?

Reply:
in realta' a questo livello non ha senso arrotondarlo, ma in caso solo quando lo si stampano i risultato. si era cambiato da 4 a 3 perche' in caso intersection_not_cog_nq nullo c'era la paura che venisse una divisone per zero. ma in realta aggiungendo 0.001 e' sufficiente arrotondarlo per 3.